web/locales/it.json

```diff
@@ -1212,9 +1212,9 @@
         },
         "RU-1": {
             "countryName": "Russia",
-            "zoneName": "Russia europea e Monti Urali"
+            "zoneName": "Russia europea ed Ural"
```

Comment:
Here can be either `Russia europea e Monti Urali` or `Russia europea ed Urali`

Reply:
Since it's about the border of the European Russia it can be referred to the Ural Mountains or the Ural river. There's a bit of ambiguity in here that have to be resolved.

Reply:
ok, let's stick with the river 👍